pkg/frontend/querymiddleware/astmapper/instant_splitting_test.go

```diff
@@ -281,59 +281,59 @@ func TestInstantSplitter(t *testing.T) {
 		// Offset operator
 		{
 			in:                   `rate({app="foo"}[3m] offset 3m)`,
-			out:                  `sum without() (__embedded_queries__{__queries__="{\"Concat\":[\"increase({app=\\\"foo\\\"}[1m] offset 5m)\",\"increase({app=\\\"foo\\\"}[1m] offset 4m)\",\"increase({app=\\\"foo\\\"}[1m] offset 3m)\"]}"}) / 180`,
+			out:                  `sum without() (__embedded_queries__{__queries__="{\"Concat\":[{\"Expr\":\"increase({app=\\\"foo\\\"}[1m] offset 5m)\"},{\"Expr\":\"increase({app=\\\"foo\\\"}[1m] offset 4m)\"},{\"Expr\":\"increase({app=\\\"foo\\\"}[1m] offset 3m)\"}]}"}) / 180`,
```

Comment:
Won't this change the format that's being sent over the network between queriers and query-frontends? Would old queriers be able to work with new query-frontends or will queries fail?

_I thought i left this comment already the other day, but apparently I didn't 🦑_

Reply:
Good point, I guess queriers and query-frontends must both be on the new version? @charleskorn WDYT?

Reply:
No: this format is only used within query-frontends. `shardedQuerier` will receive the `EmbeddedQuery` and send the embedded expression (from `Expr`) to queriers. 

So queriers will receive exactly the same thing as before, so there should be no backwards or forwards compatibility issues.